xml/ay_bigfile.xml

```diff
@@ -16196,24 +16196,21 @@ echo -n 100
      where &yast; can take over with the main control file. Currently,
      the source medium is automatically discovered, which in some cases
      makes it possible to initiate the auto-install process without giving
-     any instructions to linuxrc.
+     any instructions to <command>linuxrc</command>.
     </para>
     <para>
-     The traditional linuxrc configuration file (<filename>info</filename>)
+     The traditional <command>linuxrc</command> configuration file (<filename>info</filename>)
      has the function of giving the client enough information about the
      installation server and the location of the sources. Usually, this file
      is not required, but it is needed in special network environments
      where DHCP and BOOTP are not used or when special kernel modules need
      to be loaded.
     </para>
     <para>
-     All linuxrc keywords can be passed to linuxrc using the kernel command
-     line. The command line can also be set when creating network bootable
-     images or it can be passed to the kernel using a specially configured
-     DHCP server in combination with Etherboot or PXE.
-     <remark>emap 2011-11-07:Obscure paragraph. What is set in the command
-     line? And what is the 'it' that can be passed to the kernel? Probably not
-     the command line itself.</remark>
+     You can pass keywords to <command>linuxrc</command> using the kernel
```

Comment:
Hm, not sure I like the new version better, because it contains a lot of juxtapositions whose relationship to each other is not apparent. Could you try to split into two sentences (according to units of thoughts)?

Reply:
Good idea. I have broken this down into one sentence per method.